### ISSUE TITLE
This image is missing a text alternative (`alt` attribute). This is a problem for people using screen readers.

### ISSUE THREAD

**Issue by kadirselcuk** [turkdevops/starter-workflows#55]
This image is missing a text alternative (`alt` attribute). This is a problem for people using screen readers.

_Originally posted by @accesslint[bot] in https://github.com/turkdevops/starter-workflows/pull/48#discussion_r569588258_
